core/src/operations/decorators/mod.rs

```diff
@@ -36,6 +36,15 @@ pub enum Decorator {
     Trace(u32),
 }
 
+#[cfg(feature = "formatter")]
+impl crate::prettier::PrettyPrint for Decorator {
+    fn render(&self) -> crate::prettier::Document {
+        use crate::prettier::*;
+
+        display(self)
```

Comment:
I usually default to the separate imports since most of the time the impls need multiple helpers, but then collapse them later if that ends up not being the case, looks like I just missed that here. I'll make it consistent with the others.